rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -40,6 +40,7 @@ nova_db_max_pool_size: 120
 nova_db_pool_timeout: 60
 nova_cpu_allocation_ratio: 2.0
 nova_ram_allocation_ratio: 1.0
+nova_console_type: "spice"
```

Comment:
that should not be in user_variables.
If you need it for rpc maas, it should be in rpcm variables. But I strongly doubt we need this.